balboa-client-jms/src/main/java/com/blist/metrics/impl/queue/MetricJmsQueue.java

```diff
@@ -64,7 +64,15 @@ private MetricJmsQueue(ConnectionFactory factory, String queueName) throws JMSEx
         this.factory = factory; // badness happens if it's GC'd
         Connection connection = factory.createConnection();
         realInstance = new MetricJmsQueueNotSingleton(connection, queueName);
-        realInstance.start();
+    }
+
+    @Override
+    public void close() throws Exception {
+        // TODO When a singleton object has two underlying implementations... that is a bad smell.
```

Comment:
I will remove the random TODO.  After looking at it closely, I do not agree with the naming and how this particular class named its static member.

As far as the MetricJmsQueueNotSingleton class, I am not a big fan of it either.  It was an implementation I walked in on and was not wanting to change the implementation/interface much.